kovri.md

```diff
@@ -9,76 +9,53 @@ Ring signatures obscure the sender.
+Users could try to hide IP addresses with Tor or a VPN; however both of these strategies have serious weaknesses.
+The Tor network has "semi-trusted" Directory Authorities which  negatively affect decentralized applications and correlation attacks are possible with VPNs, making it easy for large attackers to de-anonymize a user's traffic.
 
-Currently based on the Invisible Internet Project's open specifications, Kovri uses both garlic encryption and garlic routing to create a private, protected overlay-network across the internet.
-This overlay-network provides Alice, Bob, Charlie, and Dave with the ability to effectively hide their geographical location and internet IP address from each other.
-Essentially, Kovri *covers* an application's internet traffic to make it anonymous within the network.
+So what can Alice, Bob, Charlie, and Dave do to mitigate these threats? Well, they could simply use Kovri!
+With Kovri, nodes will no longer be able to see IP addresses, making passive surveillance tactics impractical.
+Furthermore, the source of solved blocks are unknown when transmitted through Kovri, which will substantially improve Monero's censorship resistance.
 
-TODO: to harsh a segue?
-
-Alice, Bob, Charlie, and Dave all have at least one thing in common: their IP addresses are exposed to each other.
-This means they all have the ability to log the geographical locations of every peer they connect to.
-Their only choice is to hide their IP address from each other.
-
-There are several ways for them to hide their IP addresses.
-They could use Tor, or they could use a trusted VPN; but both of these strategies have weaknesses.
-
-The Tor network has "semi-trusted" Directory Authorities which give a handful of Tor node operators overreaching influence into network consensus.
-Network consensus ultimately determines who is allowed to relay traffic on the Tor network based on the views of the Authorities.
-A design flaw that has yet to be fixed, Tor's Directory Authorities negatively affect decentralization and thus negatively affect decentralized applications.
+As you can see, Kovri is a software that allows users to send transactions more privately and anonymously than ever before.
+Let's next go over some technical details to have a better understanding of how Kovri works under the hood. 
 
-Another option of using a trusted VPN brings up the issue of trivial bandwidth correlation; making it easy for 3-letter agencies to de-anonymize a user's traffic.
-VPNs also adhere to local laws in order to stay in business; so Alice, Bob, Charlie, and Dave would always run the risk of having their user data given to a government whose policies they doesn't agree with.
-Most importantly, Trusted VPNs don't protect Bob from Alice knowing who Bob is, or Dave from Charlie, and so on. This means that no-one is truly hidden from one another when using a VPN.
+## 5) Kovri and Technical Attributes
```

Comment:
I defer to others on how it should fit into this section, but I think it should be explicitly stated that Kovri allows for the transmitting of any message over the internet, not just Monero.  I don't know if we want to generalize this point or be specific with examples.  Examples may be easier, like chatting over IRC or visiting websites on the I2P network.